lib/users/index.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const config = require('config');
+const contentTypes = require('punchcard-content-types');
+const userRoutes = require('./routes');
+const fs = require('fs');
+const yaml = require('js-yaml');
+const knex = require('knex')(config.knex);
+
+/*
+  Checks for the existence of a `users` table
+
+  ## TODO
+  * This will change (or go away) when we have a table-creation process
+
+  @returns {string} process message
+ */
+const checkUserTable = () => {
+  return knex.schema.hasTable('users')
+    .then(exists => {
+      if (exists) {
+        return 'table "users" EXISTS';
+      }
+
+      return knex.schema.createTable('users', table => {
+        table.uuid('id');
+        table.string('email').unique();
```

Comment:
This flag currently doesn't exist in the schema creation, but shouldn't be too hard to add